hazelcast/proxy/cp/atomic_long.py

```diff
@@ -145,13 +146,18 @@ def alter(self, function):
 
         Args:
             function (hazelcast.serialization.api.Portable or hazelcast.serialization.api.IdentifiedDataSerializable):
-                The function applied to the currently stored value.
+                The function that alters the currently stored value.
 
         Returns:
             hazelcast.future.Future[None]:
         """
+        check_not_none(function, "Function cannot be None")
         function_data = self._to_data(function)
         codec = atomic_long_alter_codec
+        # 1 means return the new value.
```

Comment:
That's a nice addition. I had to look at the codec and server-side code to understand what `1`/`0` means. 👍